src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -279,7 +279,7 @@ public override Task OnDisconnectedAsync(HubConnectionContext connection)
             {
                 foreach (var subscription in redisSubscriptions)
                 {
-                    _logger.LogInformation("Unsubscribing from channel: {channel}", subscription);
+                    _logger.Unsubscribe(subscription);
```

Comment:
We are losing the ability to distinguish before unsubscribing from channel and from groupChannel. This can be deduced based on the channel name but I wonder if this is good enough.

Reply:
Can you share log outputs from before and after the change?

Reply:
Considering the fact that we're changing these to Trace now so they are less visible, and that the channel name will be `hubname.group.groupname` e.g. `ChatHub.group.Admins` so it is pretty easy to tell what the group channels are. I don't think it matters that we lose the "unsubscribe from group" log.

Reply:
Sounds good.